aldryn_cms/urls.py

```diff
@@ -3,8 +3,12 @@
 
 from cms.sitemaps import CMSSitemap
 
+from .views import check_uninstall_ok
+
 
 urlpatterns = [
+    # Not sure if we should hardcode admin path like this
+    url(r'^admin/~cmscloud-api/check-uninstall/$', check_uninstall_ok),
```

Comment:
url is the same, I'm not a fan of it but not sure if it was needed given that we support both base projects